app/slack-responses/members.ts

```diff
@@ -24,10 +24,11 @@ class MemberOverflowActions implements Overflow {
   action_id: string;
   options: Option[];
   constructor(member: MemberResponse, initiative: InitiativeResponse) {
-    this.action_id = MemberAction.UPDATE_MEMBERSHIP;
+    this.action_id = MemberAction.UPDATE_INITIATIVE;
     const changeMembership = new ChangeMembershipOption(member, initiative);
     const remove = new RemoveOption(member, initiative);
-    this.options = [changeMembership, remove];
+    const edit = new EditInitiativeOption(initiative);
+    this.options = [changeMembership, remove, edit];
```

Comment:
i think it's confusing to have the edit initiative button be here at the member level

what do you think about replacing the status dropdown with an edit button, and adding a status dropdown to the edit overlay?  i think that would be more clear ux